README.md

```diff
@@ -1,15 +1,14 @@
 # pydgraph
 
-This is the official Dgraph database client implementation for Python (Python >= v2.7 and >= v3.5), using [gRPC][grpc].
+This is the official Dgraph database client implementation for Python (>= v3.5), using [gRPC][grpc].
```

Comment:
Let's say >=3.11, since this is what is in the `pyproject.toml` file, and what we are testing on Ci.

Reply:
Done.